ThunderCloud/gunzip.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  gunzip.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 07/04/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import Foundation
+import zlib
```

Comment:
Seems like a good find! Definitely an easier to read than the previous code. I haven't really looked at this libraries docs and the below seems reasonable as I read through.